build/BuildFunctions.psm1

```diff
@@ -224,8 +222,30 @@ function Merge-ExtensionPackage([string[]]$tasks){
         }
     }
     $jobs | Wait-Job | Receive-Job -Wait -AutoRemoveJob
-    Write-Verbose "Running: tfx extension create --output-path bin"
-    tfx extension create --manifestGlobs **/manifest.json --output-path bin
+    $tfxArgs = "extension", "create", "--manifestGlobs", "**/manifest.json", "--output-path", "bin"
+    if ($publisher) {
+        $tfxArgs += "--publisher", $publisher
+    }
+    $overrides = @{}
+    if ($publisher) {
+        $tfxArgs += "--publisher", $publisher
+    }
+    if ($version) {
+        $overrides["version"] = $version
+    }
+    if ($overrides.Count -gt 0) {
+        $overridesFile = [System.IO.Path]::GetTempFileName()
```

Comment:
Maybe a comment or pull this into it's own function?

Reply:
Done.